lib/atom-maven.js

```diff
@@ -9,7 +9,9 @@ const mvn = require('./maven-utils');
 module.exports = {
 
 	cpConfigFileName: '.classpath',
-
+	/**
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Missing JSDoc @returns for function.](https://www.codacy.com/app/connor-bray/atom-maven/file/2737259317/issues/source?bid=3352650&fileBranchId=3352663#l12)